V1/utils/config.py

```diff
@@ -26,18 +26,46 @@ def load_config(path_to_config = './config.json'):
     data = json.loads(data)
     return data
 
+def create_deadline(deadline_value):
```

Comment:
The name is not serving the purpose. The method is used for input validation, not creation